.infrastructure/daemonset.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+ name: kube2py
+ namespace: mateapp
+ labels:
+   app: kube2py
+spec:
+ selector:
+   matchLabels:
+     app: mate-server
```

Comment:
The label 'app: mate-server' in the selector does not match the label 'app: kube2py' used in the metadata. Ensure that the labels in the selector match those in the pod template to avoid issues with pod management.